app/models/project.rb

```diff
@@ -12,8 +12,9 @@
 class Project < ActiveRecord::Base
   has_many :configs
   has_many :metric_samples
-
   has_and_belongs_to_many :users
+  has_many :ownerships
```

Comment:
Yea I think according to the rails doc that's how you implement a `has_and_belongs_to_many` relationship using `has_many`